app/scripts/modules/kubernetes/src/v2/manifest/rollout/RollingRestart.tsx

```diff
@@ -0,0 +1,55 @@
+import { module } from 'angular';
+
+import * as React from 'react';
+import { react2angular } from 'react2angular';
+
+import { Application, ManifestWriter, ReactInjector } from '@spinnaker/core';
+
+import { IKubernetesServerGroupManager } from 'kubernetes/v2/serverGroupManager';
+
+interface IRollingRestartProps {
+  application: Application;
+  serverGroupManager: IKubernetesServerGroupManager;
+}
+
+interface IRollingRestartParameters {
+  account: string;
+  cloudProvider: string;
+  location: string;
+  manifestName: string;
+}
+
+function RollingRestart({ application, serverGroupManager }: IRollingRestartProps) {
+  function rollingRestart() {
+    const rollingRestartParameters: IRollingRestartParameters = {
+      account: serverGroupManager.account,
```

Comment:
I don't think so, because [`IKubernetesServerGroupManager`](https://github.com/spinnaker/deck/blob/master/app/scripts/modules/kubernetes/src/v2/serverGroupManager/IKubernetesServerGroupManager.ts#L3) extends [`IServerGroupManager`](https://github.com/spinnaker/deck/blob/master/app/scripts/modules/core/src/domain/IServerGroupManager.ts#L10) which has `account` and `name`. In general for these resource interfaces, I think only the cloud provider-specific fields are included on the cloud provider specific implementations. But lmk if I am missing something!

Reply:
Nope, you're completely correct---I missed that!